test/Identity.IntegrationTest/Endpoints/IdentityServerTests.cs

```diff
@@ -0,0 +1,298 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Bit.Core.Enums;
+using Bit.Core.Models.Api.Request.Accounts;
+using Bit.Core.Utilities;
+using Bit.Test.Common.ApplicationFactories;
+using Bit.Test.Common.Helpers;
+using Microsoft.AspNetCore.Http;
+using Microsoft.EntityFrameworkCore;
+using Xunit;
+
+namespace Bit.Identity.IntegrationTest.Endpoints
+{
+    public class IdentityServerTests : IClassFixture<IdentityApplicationFactory>
+    {
+        private readonly IdentityApplicationFactory _factory;
+
+        public IdentityServerTests(IdentityApplicationFactory factory)
+        {
+            _factory = factory;
+        }
+
+        [Fact]
+        public async Task WellKnownEndpoint_Success()
+        {
+            var context = await _factory.Server.GetAsync("/.well-known/openid-configuration");
+
+            using var body = await AssertHelper.ResponseIsAsync<JsonDocument>(context);
+            var root = body.RootElement;
+            Assert.Equal(JsonValueKind.Object, root.ValueKind);
+            AssertHelper.AssertJsonProperty(root, "issuer", JsonValueKind.String);
+
+            var scopesSupported = AssertHelper.AssertJsonProperty(root, "scopes_supported", JsonValueKind.Array)
+                .EnumerateArray()
+                .Select(je => je.GetString());
+
+            Assert.Contains("api", scopesSupported);
+            Assert.Contains("api.push", scopesSupported);
+            Assert.Contains("api.licensing", scopesSupported);
+            Assert.Contains("api.organization", scopesSupported);
```

Comment:
api.installation now too right?

Reply:
Do you consider the order to be a breaking change or can I achieve this by asserting the amount of items and keep the rest as Contains checks?